tools/installer/requirements.txt

```diff
@@ -1 +1,2 @@
 mutagen==1.31
+ensurepip
```

Comment:
This list is for things that get installed into the SDK environment when building the windows and OSX frozen versions of Exaile, and is not used for any other purpose. The only things that can go here are python modules that can be installed via pip (eg, exist on pypi.python.org) on Windows and OSX. 

* ensurepip cannot be installed by pip, it is a builtin module
* lxml has binary dependencies, should not be on this list
* BeautifulSoup... probably cannot be installed
* cairo cannot be installed by pip, and should not be in this list
* pyHook... I don't know the answer to that
* Not sure what grp is. Does DAAP work?

I don't think its worth the time/effort to autogenerate this list, as it's not a particularly long list.